src/main/java/mekanism/common/block/states/BlockStateBasic.java

```diff
@@ -210,6 +218,14 @@ public boolean hasRotations() {
         public boolean hasActiveTexture() {
             return activable;
         }
+        
+        public boolean isOpaqueCube() {
+            if(meta == 0 && !loaded) {//This is needed to prevent Security Desks and Structural Glass from getting Opaque somhow.
```

Comment:
I'd much rather you solved this properly

Reply:
I tried to get it without that but than all blocks are Opaque idk why.

Reply:
Please define "all blocks are opaque" - how do you know this? what observable effect does it have?

Reply:
Structural Glass and Security Desk are getting rendered wrong.
Like black and the glass is doing too much shadows.

Reply:
alright, so have you set a breakpoint here (inside your `if` statement), and see what is calling it?

Reply:
yeah i have tried that and its the Block init.

Reply:
ok i got that now i am trying to fix the explosion resistance of all BlockBasics but Refined Obsidian.

Reply:
OK i got it.